### DISCUSSION TITLE
Misaligned Data Between /forecast/hourly and the "raw" data

### DISCUSSION THREAD

**Discussion by jeffclark** [weather-gov/api#323] [Q&A]
I'm looking for forecast data by the hour, but am getting conflicting information between the raw data at `gridpoints/CWA/gridX,gridY/` and the hourly forecast generated at `gridpoints/CWA/gridX,gridY/forecast/hourly`

**Examples from 12:00:00 2-July**
The `temperature` at: https://api.weather.gov/gridpoints/BOX/81,72
Returns: 
`{ "validTime": "2020-07-02T12:00:00+00:00/PT1H", "value": 18.333333333333332 }`
(Converting to F is 65)

For the same time period, the `temperature` at: https://api.weather.gov/gridpoints/BOX/81,72/forecast/hourly
Returns: `74`

(Sitting outside, it's definitely closer to 74 than 65!)

The skycover data at https://api.weather.gov/gridpoints/BOX/81,72
Returns: 
`{ "validTime": "2020-07-02T12:00:00+00:00/PT1H", "value": 74}`
Which I believe means that there is 74% cloud coverage.

For the same time period, the `shortForecast` at: https://api.weather.gov/gridpoints/BOX/81,72/forecast/hourly
Returns: `Sunny`

(Sitting outside, it's certainly sunny and not 74% cloud cover! A single small cloud in the sky!)

Am I misinterpreting the "raw" data incorrectly? Should I expect these two data points align with each other?

**Comment by StephenClouse** [comment]
I think you are missing the difference in time zone offset.

The raw data is in UTC (+00:00).

The forecasts are in local time (EDT -04:00 in this case).

`2020-07-02T12:00:00+00:00` and `2020-07-02T08:00:00-04:00` are the same time instant.

So if you are, as I suspect, looking at this period in the hourly forecast data:

`            {
                "number": 2,
                "name": "",
                "startTime": "2020-07-02T12:00:00-04:00",
                "endTime": "2020-07-02T13:00:00-04:00",
                "isDaytime": true,
                "temperature": 74,
                "temperatureUnit": "F",
                "temperatureTrend": null,
                "windSpeed": "5 mph",
                "windDirection": "S",
                "icon": "http://localhost/icons/land/day/few?size=small",
                "shortForecast": "Sunny",
                "detailedForecast": ""
            },`

Then in the grid you should be looking for data corresponding to time `2020-07-02T16:00:00+00:00`.

I think you will find these data align properly.

temperature: `                        {
                            "validTime": "2020-07-02T16:00:00+00:00/PT3H",
                            "value": 23.333333333333332
                        },`

skyCover: `                        {
                            "validTime": "2020-07-02T15:00:00+00:00/PT4H",
                            "value": 22
                        },`

**Comment by jeffclark** [comment]
@StephenClouse Ouch, good eye. That's embarrassing. Thanks for the help!

**Comment by StephenClouse** [comment]
@jeffclark Don't worry, you're not the first :)
